Makefile

```diff
@@ -130,4 +131,10 @@ restart-argus:
 # ADMIN
 # pushes appimage to storage bucket.
 push-package:
-	gsutil cp $(BIN_DIR)/viam-csi-$(PACK_TAG)-aarch64.AppImage gs://viam-csi
+	gsutil cp $(BIN_DIR)/viam-csi-$(PACK_TAG)-aarch64.AppImage gs://packages.viam.com/apps/csi-camera/
+
+# pushes base docker image to dockerhub.
+push-base:
```

Comment:
Just saw this push and wanted to make a note. We use ghcr.io (the github container registry) for our other images, like canon. I think that would be preferable to use here as well, rather than an individual user's account on dockerhub, as it more easily integrates with our github permissions and teams.